docs/source/index.rst

```diff
@@ -54,11 +54,11 @@ By default, Sycamore works with the Aryn Partitioning Service to process PDFs. y
 
 More Resources
 --------------------
-- Visit the Aryn DocParse and DocPrep Documentation: https://docs.aryn.ai/introduction
+- Visit Aryn DocParse and DocPrep Documentation: https://docs.aryn.ai/introduction
 - Join the Aryn / Sycamore Slack workspace: https://join.slack.com/t/sycamore-ulj8912/shared_invite/zt-23sv0yhgy-MywV5dkVQ~F98Aoejo48Jg
-- Watch the intro video to the Aryn Partitioning Service: https://www.aryn.ai/?name=ArynPartitioningService_Intro
-- Sign up for the Aryn Partitioning Service: https://aryn.ai/get-started
-- Use the Aryn Playground to experiment with the Partitioning Service: https://play.aryn.cloud/partitioning
+- Watch the intro video to Aryn DocParse (formerly known as the Aryn Partitioning Service): https://www.aryn.ai/?name=ArynPartitioningService_Intro
+- Sign up for Aryn DocParse: https://aryn.ai/get-started
+- Use the Aryn Playground to experiment with DocParse: https://play.aryn.cloud/partitioning
```

Comment:
You can link to the DocPrep video too: https://www.youtube.com/watch?v=OMjLQ_0bSgU